src/ProfilePage/ProfileBadges.jsx

```diff
@@ -0,0 +1,160 @@
+const accountId = props.accountId;
+if (!accountId || !context.accountId) return null;
+
+const GRAPHQL_ENDPOINT = props.GRAPHQL_ENDPOINT || "https://near-queryapi.api.pagoda.co";
+State.init({
+  verifications: null,
+  loadingState: "none",
+});
+
+const fetchGraphQL = (operationsDoc, operationName, variables) => {
+  return asyncFetch(`${GRAPHQL_ENDPOINT}/v1/graphql`, {
+    method: "POST",
+    headers: { "x-hasura-role": "dataplatform_near" },
+    body: JSON.stringify({
+      query: operationsDoc,
+      variables: variables,
+      operationName: operationName,
+    }),
+  });
+};
+
+const createQuery = () => {
+  return `
+query GetVerifications {
+    dataplatform_near_verifications_account(where: {account_id: {_eq: "${accountId}"}}) {
+      human_provider
+      human_valid_until
+      human_verification_level
+      kyc_provider
+      kyc_valid_until
+      kyc_verification_level
+      social_trust_provider_1
+      social_trust_provider_2
+      social_trust_score_1
+      social_trust_score_2
+      account_id
+    }
+}
+`;
+};
+
+const loadItems = () => {
+  const queryName = "GetVerifications";
+  return fetchGraphQL(createQuery(), queryName).then((result) => {
+    if (result.status === 200 && result.body) {
+      if (result.body.errors) {
+        console.log("error:", result.body.errors);
+        return;
+      }
+      let data = result.body.data;
+      if (data) {
+        const newItems = data.dataplatform_near_verifications_account;
+        if (newItems && newItems[0]) {
+          State.update({
+            verifications: newItems[0],
+            loadingState: "loaded",
+          });
+        }
+      }
+    }
+  });
+};
+
+if (!state.loadingState || state.loadingState === "none") {
+  State.update({ loadingState: "loading" });
+  state.loadingState = "loading";
+  loadItems();
+  return <p>Loading verifications...</p>;
+}
+if (state.loadingState === "loaded" && !state.verifications) {
+  return null;
+}
+
+const VerificationText = styled.div`
+  display: inline;
+  position: relative;
+  top: 1px;
+  font-size: 14px;
+  color: ${(props) => (props.secondary ? "#717069" : "black")};
+`;
+if (state.verifications) {
+  return (
+    <div>
```

Comment:
Let's make a <Wrapper> here instead of simple div:
```
const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  gap: 5px;
`;
```